### PR TITLE
View certificates for OEM

### PR DESCRIPTION
**Pull request by jonmattgray**
- [x] View and download order certificates for OEM once the order has at least `status: accepted`.
- [x] Styling cleanup and removal of empty components

Certificates always show, can't click to show/hide.

----------**BEFORE**----------
image

----------**AFTER**----------
image

doesn't show before `accepted`
image

after accepting
image

### CODE REVIEW

File: src/customer/components/CertificationRow.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+  return (
+    <Paper elevation={0} className={classes.root}>
+      <Grid container>
+        <Grid item xs={1} className={classes.rowItem}>
+          {file ? (
+            <img src={tick} className={classes.icon} />
+          ) : (
+            <img src={pending} className={classes.icon} />
+          )}
```
Comment (darrylmorton-digicatapult) [inline src/customer/components/CertificationRow.js:50]: ``

File: src/customer/components/CertificationRow.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        </Grid>
+        <Grid
+          item
+          xs={7}
+          className={`${file ? '' : classes.greyText} ${classes.rowItem}`}
+        >
+          <Typography>{description}</Typography>
+        </Grid>
+        {file && (
```
Comment (darrylmorton-digicatapult) [inline src/customer/components/CertificationRow.js:59]: might be worth combining line 52-58 into this condition block:

changing `{file &&` to `{file ?`
add line 52-58 here without className

the `:` condition just contains
`<Grid
          item
          xs={7}
          className={${classes.greyText} ${classes.rowItem}}
        >
          <Typography>{description}</Typography>
        </Grid>`

**Review by darrylmorton-digicatapult — CHANGES_REQUESTED** [review]

File: src/customer/components/CertificationRow.js (hunk 1 above)
Comment (jonmattgray) [reply src/customer/components/CertificationRow.js]: changed, thanks

File: src/customer/components/CertificationRow.js (hunk 2 above)
Comment (jonmattgray) [reply src/customer/components/CertificationRow.js]: changed

File: src/am/components/Orders/CertificationRow.js [hunk 3]
```diff
@@ -99,54 +99,56 @@ const CertificationRow = ({
...
+            <Grid item xs={2} className={classes.rowItem}>
+              <Grid
+                container
+                direction="column"
+                className={classes.dnd}
+                {...getRootProps()}
+              >
+                <input {...getInputProps()} />
+                <Grid item xs={6}>
+                  <img src={cloud} className={`${classes.icon}`} />
```
Comment (darrylmorton-digicatapult) [inline src/am/components/Orders/CertificationRow.js:141]: only className={classes.icon} /> needed

**Review by darrylmorton-digicatapult — APPROVED** [review]

**Review by dblane-digicatapult — APPROVED** [review]

**Review by danielcooperxyz — APPROVED** [review]

**Pull request merged**